src/parser.rs

```diff
@@ -165,11 +165,13 @@ impl Library {
         }
 
         let glib_name = attrs.get("type-name").unwrap_or(name);
+        let get_type_func_name = attrs.get("get-type").unwrap_or("None");
```

Comment:
This should use the same error handling as `"name"`:
```rust
let get_type_func_name = try!(
    attrs.get("get-type").ok_or_else(|| error!("Missing get-type attribute", parser)));
```
and located above the loop so the `parser` would report the correct position.